aspnetcore/blazor/globalization-localization.md

```diff
@@ -873,6 +873,56 @@ To further understand how the Blazor framework processes localization, see the [
 
 :::zone-end
 
+## Shared resources
+
+To create localization shared resources, adopt the following approach. For additional information, see the shared resources guidance in the <xref:fundamentals/localization#make-the-apps-content-localizable> article.
```

Comment:
For additional information, see the shared resources guidance in the <xref:fundamentals/localization#make-the-apps-content-localizable> article.

IMHO this should moved at the end